examples/full/src/components/header/index.js

```diff
@@ -2,6 +2,7 @@ import { h, Component } from 'preact';
 import { Link } from 'preact-router/match';
 import style from './style';
 
+// eslint-disable-next-line
```

Comment:
@reznord is that in the plans? I could help with that! Wanted to learn how to make eslint plugins and configs

_Sent from my Bq Aquaris M5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_